docs/reference/self-hosted.md

```diff
@@ -0,0 +1,83 @@
+# Self-Hosted Deployment with Helm
+
+This reference provides an overview of the official [Logfire Helm chart](https://github.com/pydantic/logfire-helm-chart). Enabling easy deployment and management of Logfire on Kubernetes.<br>
+This chart is included in our Enterprise plan, [Contact us](https://pydantic.dev/contact) for details
+
+### Key Benefits
+
+* **Simplified Deployment:** Install and manage the entire application stack with a single command.
+* **Flexible Configuration:** Easily adjust resource allocation, ingress settings, and authentication to your needs.
+* **Production-Ready Defaults:** Built-in settings for high availability, resource limits, and health checks.
+* **Repeatable & Versioned:** Manage your application deployment as code, ensuring consistency across environments.
+* **Compliance Friendly:** Easily aligns with internal security standards and regulatory requirements by leveraging your organization's own infrastructure.
+---
+
+## Prerequisites
+
+Before deploying, you will need the following:
+
+* **Kubernetes Cluster:** A running Kuberentes cluster
+* **External Resources:** A production-ready PostgreSQL database and object storage service (e.g., AWS S3, Google Cloud Storage, Azure Blob Storage).
+* **Image Pull Secret:** Obtain credentials for accessing our private container images by contacting **sales@pydantic.dev**.
+
+---
+
+## Configuration Overview
+
+Deploying Logfire successfully involves configuring essential parameters within your ```values.yaml``` file. Below is an overview of the topics you need to go over
+
+### 1. Image Pull Secret
+
+After receiving credentials from us, create a docker-registry secret in your Kubernetes cluster and reference its name in your ```values.yaml```.
+
+### 2. Ingress & Hostname
+
+Set ```ingress.hostname``` and ingress settings to expose Logfire at your desired URL.
+
+### 3. Authentication (Dex)
+
+The chart uses Dex as its identity service. You must configure at least one connector (like GitHub, OIDC, LDAP, etc.) so users can log in. You can find a full list of supported connectors in the [Dex documentation](https://dexidp.io/docs/connectors/).
+
+### 4. External Services
+
+Supply connection details for your PostgreSQL database and object storage.<br>
+**Note**: The chart doesn't deploy production-grade databases or storage by default.
```

Comment:
I think we want another sentence here explaining the implication / so what? Something like "we can work with your team to tune the database to meet your data volume requirements"